xml/installation-installation-configure_lbaas.xml

```diff
@@ -233,482 +233,363 @@
-| security_group_id | 75343a54-83c3-464c-8773-802598afaee9 |
-| tenant_id         | 4b31d0508f83437e83d8f4d520cda22f     |
-+-------------------+--------------------------------------+</screen>
+<screen>&prompt.ardana;openstack router create --centralized lb_router2
++-------------------------+--------------------------------------+
+| Field                   | Value                                |
++-------------------------+--------------------------------------+
+| admin_state_up          | UP                                   |
+| availability_zone_hints |                                      |
+| availability_zones      |                                      |
+| created_at              | 2019-10-23T13:30:21Z                 |
+| description             |                                      |
+| distributed             | False                                |
+| external_gateway_info   | None                                 |
+| flavor_id               | None                                 |
+| ha                      | False                                |
+| id                      | 1c9949fb-a500-475d-8694-346cf66ebf9a |
+| name                    | lb_router2                           |
+| project_id              | de095070f242416cb3dc4cd00e3c79f7     |
+| revision_number         | 0                                    |
+| routes                  |                                      |
+| status                  | ACTIVE                               |
+| tags                    |                                      |
+| updated_at              | 2019-10-23T13:30:21Z                 |
++-------------------------+--------------------------------------+</screen>
    </step>
    <step>
     <para>
-     Create TCP port 80 rule.
+     Add the subnet to the router and connect it to the external GW
     </para>
-<screen><?dbsuse-fo font-size="0.70em"?>&prompt.ardana;openstack security group rule create --protocol tcp --dst-port 80 lb_secgroup1
-+-------------------+--------------------------------------+
-| Field             | Value                                |
-+-------------------+--------------------------------------+
-| direction         | ingress                              |
-| ethertype         | IPv4                                 |
-| id                | 10a76cad-8b1c-46f6-90e8-5dddd279e5f7 |
-| port_range_max    | 80                                   |
-| port_range_min    | 80                                   |
-| protocol          | tcp                                  |
-| remote_group_id   |                                      |
-| remote_ip_prefix  |                                      |
-| security_group_id | 75343a54-83c3-464c-8773-802598afaee9 |
-| tenant_id         | 4b31d0508f83437e83d8f4d520cda22f     |
-+-------------------+--------------------------------------+</screen>
+<screen>&prompt.ardana;openstack router add subnet lb_router1 lb_subnet2</screen>
```

Comment:
lb_router1 should be lb_router2

Reply:
fixed this in the latest bit, I did not update the zip file (yet)